apps/ensindexer/src/plugins/multi-chain/ReverseResolver.ts

```diff
@@ -0,0 +1,49 @@
+import { ponder } from "ponder:registry";
```

Comment:
Wondering if this file should be moved to `apps/ensindexer/src/handlers/` directory, so the plugins directory can only include plugin directories?